src/styles/navbar.scss

```
@@ -0,0 +1,64 @@
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+header {
```

Use class selector for header
```suggestion
.header {
```